acceptance/serverless.yml

```diff
@@ -1,4 +1,4 @@
-service: qa-events-agent
```

Comment:
be sure to delete the old service after this deploys (by having an old version of the code running and run `sls remove`